production-elasticsearch/jaeger-production-template.yml

```diff
@@ -0,0 +1,187 @@
+#
```

Comment:
I think we should include elasticsearch in the name.

Reply:
done